packages/graphiql/vite.config.mts

```diff
@@ -0,0 +1,69 @@
+import { defineConfig } from 'vite';
+import packageJSON from './package.json';
+import dts from 'vite-plugin-dts';
+import commonjs from 'vite-plugin-commonjs';
+
+const umdConfig = defineConfig({
+  define: {
+    'process.env.NODE_ENV': `"${process.env.NODE_ENV}"`,
+  },
+  // To bundle `const { createClient } = require('graphql-ws')` in `createWebsocketsFetcherFromUrl` function
+  plugins: [commonjs()],
+  build: {
+    minify: true,
+    sourcemap: true,
+    emptyOutDir: false,
+    lib: {
+      entry: 'src/cdn.ts',
+      // 👇 The name of the exposed global variable. Required when the formats option includes umd or iife
+      name: 'GraphiQL',
+      fileName: 'index',
+      formats: ['umd'],
+    },
+    rollupOptions: {
+      external: ['react', 'react-dom'],
+      output: {
+        globals: {
+          react: 'React',
+          'react-dom': 'ReactDOM',
+        },
+      },
+    },
+  },
+});
+
+const esmConfig = defineConfig({
+  build: {
+    minify: false,
+    sourcemap: true,
+    lib: {
+      entry: 'src/index.ts',
+      fileName: 'index',
+      formats: ['cjs', 'es'],
```

Comment:
```suggestion
      formats: ['cjs', 'es'],
```
do we really need a bundle of `cjs` version for the next major? or just left esm?

Reply:
It's quite a big step to get rid of cjs, if it doesn't cost you much to keep outputting cjs?

Reply:
with dropping `cjs` we can bundle `graphql-toolkit` to esm only too and fix https://github.com/graphql/graphiql/issues/3668